sentinel-adapter/sentinel-web-servlet/README.md

```diff
@@ -55,7 +55,7 @@ For REST APIs, you have to clean the URL resource (e.g. `/foo/1` and `/foo/2` ->
 the amount of context and resources will exceed the threshold.
 
 If you need to exclude some URLs (that should not be recorded as Sentinel resources), you could also
-leverage the `UrlCleaner` interface. You may unify the unwanted URLs to the empty string `""`,
+leverage the `UrlCleaner` interface. You may unify the unwanted URLs to the empty string `""` or null,
```

Comment:
This PR is for Spring WebFlux adapter only, so it might be better to make this change in other PRs.

Reply:
Ok, I will try to change this in another PR.